test/fuzz_group.cpp

```diff
@@ -434,25 +437,59 @@ void parse_and_apply_instructions(std::string& in, const std::string& path, util
+                             << name << "\", " << (nullable ? "true" : "false") << ");\n";
+                    }
+                    g.get_table(table_ndx)->add_column(type, name, nullable);
+                }
+                else {
+                    bool subnullable = (get_next(s) % 2 == 0);
+                    if (log) {
+                        *log << "{\n"
+                             << "DescriptorRef subdescr;\n"
+                             << "g.get_table(" << table_ndx << ")->add_column(type_Table, \"" << name << "\", "
+                             << (nullable ? "true" : "false") << ", &subdescr);\n"
+                             << "subdescr->add_column(type_Int, \"integers\", nullptr, "
+                             << (subnullable ? "true" : "false") << ");\n"
+                             << "}\n";
+                    }
+                    DescriptorRef subdescr;
+                    g.get_table(table_ndx)->add_column(type, name, subnullable, &subdescr);
+                    subdescr->add_column(type_Int, "integers", nullptr, subnullable);
                 }
-                g.get_table(table_ndx)->add_column(type, name, nullable);
             }
             else if (instr == INSERT_COLUMN && g.size() > 0) {
                 size_t table_ndx = get_next(s) % g.size();
                 size_t col_ndx = get_next(s) % (g.get_table(table_ndx)->get_column_count() + 1);
                 DataType type = get_type(get_next(s));
                 std::string name = create_column_name(s);
-                bool nullable = (type == type_Mixed || type == type_Table) ? false : (get_next(s) % 2 == 0);
-                if (log) {
-                    *log << "g.get_table(" << table_ndx << ")->insert_column(" << col_ndx << ", DataType("
-                         << int(type) << "), \"" << name << "\", " << (nullable ? "true" : "false") << ");\n";
+                bool nullable = (type == type_Mixed) ? false : (get_next(s) % 2 == 0);
+                if (type != type_Table) {
+                    if (log) {
+                        *log << "g.get_table(" << table_ndx << ")->insert_column(" << col_ndx << ", DataType("
+                             << int(type) << "), \"" << name << "\", " << (nullable ? "true" : "false") << ");\n";
+                    }
+                    g.get_table(table_ndx)->insert_column(col_ndx, type, name, nullable);
+                }
+                else {
+                    bool subnullable = (get_next(s) % 2 == 0);
+                    if (log) {
+                        *log << "{\n"
+                             << "DescriptorRef subdescr;\n"
+                             << "g.get_table(" << table_ndx << ")->insert_column(" << col_ndx << ", type_Table, "
+                             << "\"" << name << "\", " << (nullable ? "true" : "false") << ", &subdescr);\n"
+                             << "subdescr->add_column(type_Int, \"integers\", nullptr, "
```

Comment:
We can ... but then the SET operation should also be expanded, and I am not sure how much benefit we would get from that.